lib/Analysis/AxisInfo.cpp

```diff
@@ -1084,9 +1084,9 @@ LogicalResult AxisInfoAnalysis::visitOperation(
 
 void AxisInfoAnalysis::visitForOpInductionVar(
     scf::ForOp op, ArrayRef<dataflow::Lattice<AxisInfo> *> argLattices) {
-  ProgramPoint programPoint(op);
-  auto lb = getLatticeElementFor(&programPoint, op.getLowerBound())->getValue();
-  auto step = getLatticeElementFor(&programPoint, op.getStep())->getValue();
+  ProgramPoint *programPoint = getProgramPointBefore(op);
```

Comment:
> For dense forward data-flow analysis and other analysis (except dense backward data-flow analysis), the program point corresponding to the original operation can be obtained by getProgramPointAfter(op)

So why do we use `getProgramPointBefore` here?

Reply:
should it be: 
```
getLatticeElementFor(op, op.getLowerBound())->getValue();
```

Reply:
@Jokeren thanks for catching, fixed.